Install instruction (#96)
flatironinstitute/nemos: docs/index.md

and [Jonathan Pillow's Cosyne 2018
tutorial](https://www.youtube.com/watch?v=NFeGW5ljUoI&t=424s).

## Installation
Run the following `pip` command in your virtual environment.

```bash
pip install git+https://github.com/flatironinstitute/nemos.git
```

> For more comprehensive instructions, including specifics for GPU users and developers, refer to [this section](installation.md).

Reviewer: ```suggestion
For more comprehensive instructions, including specifics for GPU users and developers, refer to [the installation page](installation.md).
```
